torch/lib/THPP/tensors/generic/THSTensor.cpp

```diff
@@ -174,21 +194,160 @@ auto THSTensor<real>::unfold(const Tensor& src, int dimension,
 }
 
 template<>
-auto THSTensor<real>::squeeze(const Tensor& src, int dimension) -> THSTensor& {
+auto THSTensor<real>::squeeze(const Tensor& src) -> THSTensor& {
   throw std::runtime_error("THSTensor::squeeze not supported");
 }
 
 template<>
+auto THSTensor<real>::squeeze1d(const Tensor& src, int dimension) -> THSTensor& {
+  throw std::runtime_error("THSTensor::squeeze1d not supported");
+}
+
+template<>
 auto THSTensor<real>::unsqueeze(const Tensor& src, int dimension) -> THSTensor& {
   throw std::runtime_error("THSTensor::unsqueeze not supported");
 }
 
+#define LAPACK_ERROR "Lapack operations not yet available for sparse tensors"
```

Comment:
Don't think we need to say "yet" here 😄 It's not that they're going to be implemented anytime soon.